uk.co.compucorp.civicrm.hrleaveandabsences/xml/schema/CRM/HRLeaveAndAbsences/AbsenceType.xml

```diff
@@ -154,5 +154,11 @@
     <type>int unsigned</type>
     <comment>The unit (months or days) of carry_forward_expiration_duration of this type default expiry</comment>
   </field>
+  <field>
+    <name>is_sick</name>
+    <label>Is Sick?</label>
+    <type>boolean</type>
+    <default>0</default>
```

Comment:
can you please add a comment? "A flag which is used to determine if this Absence Type can be used for a Sickness Request" (You'll need to update the DAO and the auto_install.sql file after that)